zuul.d/molecule.yaml

```diff
@@ -109,6 +109,16 @@
       - ^molecule-requirements.txt
       - ^ci_framework/roles/local_env_vm/(?!meta).*
       - ^ci/playbooks/molecule.*
+- job:
+    name: cifmw-molecule-openshift_setup
+    parent: cifmw-molecule-base
```

Comment:
OR - add the job to the list here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/scripts/create_role_molecule#L4

it will get CRC.